catalog/app/containers/SearchResults/index.js

```diff
@@ -29,10 +35,18 @@ export class SearchResults extends React.PureComponent { // eslint-disable-line
     return (
       <div>
         <h1><FormattedMessage {...messages.header} /></h1>
-        <PackageList
-          emptyMessage={<FormattedMessage {...messages.empty} />}
-          packages={response.packages}
-        />
+        <Pagination
+          items={response.packages}
+          perPage={perPage}
+          page={page}
+          onPageChange={setPage}
+          onPerPageChange={setPerPage}
+        >{({ items }) =>
```

Comment:
hard to read as written; brackets should be on their own line and same for arrow expression args

Reply:
adjusted